packages/@magic-ext/connect/src/index.ts

```diff
@@ -0,0 +1,26 @@
+import { Extension } from 'magic-sdk';
+import { UserInformation } from './types';
+
+export class ConnectExtension extends Extension.Internal<'connect', any> {
+  name = 'connect' as const;
+  config: any = {
+    mc: true,
+  };
+
+  public showWallet() {
+    const requestPayload = this.utils.createJsonRpcRequestPayload('mc_wallet');
```

Comment:
Can we put payload methods strings to an enum?

Reply:
Updated